openwisp_ipam/migrations/__init__.py

```diff
@@ -14,13 +21,10 @@ def assign_permissions_to_groups(apps, schema_editor):
     admins_can_manage = ['subnet', 'ipaddress']
     operators_can_manage = ['ipaddress']
     manage_operations = ['add', 'change', 'delete', 'view']
-    Group = apps.get_model('openwisp_users', 'Group')
+    Group = get_swapped_model(apps, 'openwisp_users', 'Group')
 
-    try:
-        admin = Group.objects.get(name='Administrator')
-        operator = Group.objects.get(name='Operator')
-    except Group.DoesNotExist:
-        return
+    admin = Group.objects.get(name='Administrator')
+    operator = Group.objects.get(name='Operator')
```

Comment:
I remove it based on this @pandafy comment here:-
 https://github.com/openwisp/openwisp-controller/pull/390#pullrequestreview-610767868

If Group does not exist, then code shouldn't fail silently. Since we want to setup these permissions, noise should be generated on error

Reply:
Makes sense! 👍🏻